bundles/org.openhab.core.ui/src/main/java/org/openhab/core/ui/internal/proxy/ProxyServletService.java

```diff
@@ -96,6 +113,28 @@
     protected ItemUIRegistry itemUIRegistry;
     protected ModelRepository modelRepository;
 
+    @Reference(cardinality = ReferenceCardinality.MULTIPLE, policy = ReferencePolicy.DYNAMIC)
+    protected void addSitemapProvider(SitemapProvider provider) {
+        sitemapProviders.add(provider);
+        provider.addModelChangeListener(this);
+    }
+
+    @Override
+    public void modelChanged(String modelName, EventType type) {
```

Comment:
Why does the class implement `ModelRepositoryChangeListener`? This method does not seem to do anything besides doing a debug log statement.

Reply:
Leftover from testing.   Removed.